htdocs/web_portal/views/user/register.php

```diff
@@ -11,25 +11,19 @@
     <div class="alert alert-warning" role="alert">
             <h3>Terms and Conditions of Account Registration</h3>
             <ul>
-               <li>Registering a GOCDB account means <b>you accept that your ID string, your basic user details and roles will be visible to other authenticated users and client-services of GOCDB</b>, including those authenticated by: </br></br>
-                  <ol>
-                    <li>a certificate issed from a Certification Authority (CA) that is registered with the <a href="https://www.igtf.net">Interoperable Global Trust Federation (IGTF).</a></li>
-                    <li>new authentication/security realms will be added here and you will be notified by email and in the portal.</li>
-                  </ol>
+               <li>By registering a GOCDB account you are agreeing to abide by the <a href="/aup.html" target="_blank" title="opens in new window">GOCDB Acceptable Use Policy and Conditions of Use <img src="/portal/img/new_window.png" alt="new window logo" class="new_window"></a>.
                </li>
-            </ul>
-            <br>
-            <ul>
-               <li>Your details are re-published by GOCDB and used by EGI for Monitoring, Accounting and for use in its data processing systems.</li>
-               <li><a href="https://wiki.egi.eu/wiki/GOCDB/data_privacy">Further details and terms/conditions of use here</a>.</li>
-               <li>If you do not provide this consent, please <b>DO NOT register</b>.</li>
+               <li>Personal data, which you provide below and that is collected when you use GOCDB, will be processed in accordance with the <a href="/privacy.html" target="_blank" title="opens in new window">GOCDB Privacy Notice <img src="/portal/img/new_window.png" alt="new window logo"  class="new_window"></a>. 
+               </li><br>
+               <li>Please read these documents before registering your account.
```

Comment:
When I first read this line, I felt like links to the documents were missing. I dont want to duplicate the links, so how about the following?

```suggestion
               <li>Please read both the above documents before registering your account.
```